src/app/exercise/exercise/exercise.base.ts

```diff
@@ -72,7 +72,8 @@ export class ExerciseBase implements OnDestroy {
   constructor(public slide: SlideComponent, private monacoConfig: MonacoConfigService) {
     this.onActiveUsubscribe = slide.onActive.filter(a => a).subscribe(() => {
       console.log('ACTIVE');
-      slide.disableShortcuts();
+
+      // slide.disableShortcuts();
```

Comment:
Hi @prestonvanloon, yes it was intentionally commented. Correct me if I'm wrong, Due to changes in shortcuts.directive.ts keydown event is only invoking function to navigate slide when focused target is body. Which allows users to still press bind keys inside the monaco editor without causing slide changes. I thought slide.disableShortcuts() function is used in exercise.base.ts to prevent this from happening.